microsite/data/plugins/opencost.yaml

```
@@ -5,6 +5,6 @@ authorUrl: https://opencost.io
 category: Monitoring
 description: OpenCost provides cloud cost monitoring for your cloud native environments.
 documentation: https://github.com/backstage/backstage/blob/master/plugins/opencost/README.md
-iconUrl: img/opencost.png
+iconUrl: https://avatars.githubusercontent.com/u/105945214?s=200&v=4
```

sorry we missed a `/` which is probably causing the issue 🙈

```suggestion
iconUrl: /img/opencost.png
```

---

Yep, adding the slash should make it work it seems